### PR TITLE
DHFPROD-8996: Verify the privileges assigned to hub-central-mapping-reader role

### PR DESCRIPTION
**Pull request by littlecherro**
#### Checklist: 
```diff
- Note: do not change the below
```

- ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
- [x] Ran cypress tests on firefox locally
  

- ##### Reviewer:

- [x] Reviewed Tests
- N/A Added to Release Wiki

### CODE REVIEW

**Review by briantang — APPROVED** [review]

File: marklogic-data-hub-central/ui/e2e/cypress/integration/login/authorization.spec.tsx [hunk 1]
```diff
@@ -212,26 +213,65 @@ describe("login", () => {
...
     });
 
+    cy.log("**Navigates to Curate and opens EntityTypeId name**");
     toolbar.getCurateToolbarIcon().click();
     curatePage.toggleEntityTypeId(entityTypeId);
     curatePage.verifyTabs(entityTypeId, "be.visible", "not.exist");
+
+    cy.log("**User cannot see Matching tab**");
+    // TO DO: change this after DHFPROD-8997 is merged
+    cy.get(`#${entityTypeId} [data-rr-ui-event-key="match"]`).should("not.exist");
```
Comment (csilva-ml) [inline marklogic-data-hub-central/ui/e2e/cypress/integration/login/authorization.spec.tsx:233]: Can we move this cy.get() to a page object file and call the method here?

**Review by csilva-ml — CHANGES_REQUESTED** [review]
Great work! Just a minor change

File: marklogic-data-hub-central/ui/e2e/cypress/integration/login/authorization.spec.tsx (hunk 1 above)
Comment (littlecherro) [reply marklogic-data-hub-central/ui/e2e/cypress/integration/login/authorization.spec.tsx]: @csilva-ml done :)

**Review by briantang — APPROVED** [review]

**Review by csilva-ml — APPROVED** [review]
Great work!!

**Pull request merged**